makeMoney/script.js

```diff
@@ -0,0 +1,32 @@
+const countInput = document.querySelector(".count-input");
+const coinTypeSelect = document.querySelector("#coin-type");
+const coinButton = document.querySelector("#coin-button");
+const coinContainer = document.querySelector(".coin-container");
+
+const addCoin = (coinType) => {
+  const coin = document.createElement("div");
+  coin.className = "coin";
+  coin.textContent = coinType;
+  coin.addEventListener("click", () => {
+    coinContainer.removeChild(coin);
+  });
+  coinContainer.appendChild(coin);
+};
+
+coinButton.addEventListener("click", () => {
+  const coinType = coinTypeSelect.value;
+  const count = parseInt(countInput.value);
+
+  if (!isNaN(count) && count > 0) {
```

Comment:
Nice error handling for an empty number input. You could also give your number input a default value so that it is always a number.